README.md

```diff
@@ -31,16 +31,24 @@ You can find a more thorough introduction [here](https://www.jakso.me/blog/kysel
 
 # Table of contents
 
+- [Kysely](#kysely)
+- [Table of contents](#table-of-contents)
```

Comment:
These two don't need to be here.

Reply:
It was added by prettier (I have it automatically run after each file save). Didn't even realize it added these 2.